kubernetes/cray-opa/templates/_policy-ingressgateway-customer-admin.tpl

```diff
@@ -44,16 +44,17 @@ original_path = o_path {
     o_path := http_request.path
 }
 
-# Whitelist Keycloak, since those services enable users to login and obtain
-# JWTs. vcs is also enabled here. Legacy services to be migrated or removed:
-#
-#     * VCS/Gitea
-#
+# Allow all access to Gitea/VCS
 allow {
-    any([
-        startswith(original_path, "/keycloak"),
-        startswith(original_path, "/vcs"),
-    ])
+    startswith(original_path, "/vcs")
```

Comment:
Modifying the admin policy just for consistency, even through it isn't used (as the admin gateway is not used) in CSM 1.2.